### PR TITLE
Creating a stub for aveiro

### PR DESCRIPTION
**Pull request by janppires**

### CODE REVIEW

File: aveiro-pt.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## Aveiro, PT
+
+### Places to visit
+
+### Things to do
```
Comment (janppires) [inline aveiro-pt.md:8]: Does it has sun?

**Comment by janppires** [comment]
:+1: this looks good :shipit: vrummmm!!

**Comment by NuriaTorne** [comment]
:+1:

**Comment by janppires** [comment]
This PR closes #67

**Pull request merged**